core/server/worker/src/test/java/alluxio/worker/block/meta/BlockMetaTest.java

```diff
@@ -49,8 +51,9 @@ public void before() throws Exception {
     mTestDirPath = mFolder.newFolder().getAbsolutePath();
     // Sets up tier with one storage dir under mTestDirPath with 100 bytes capacity.
     TieredBlockStoreTestUtils.setupConfWithSingleTier(null, TEST_TIER_ORDINAL,
-        TEST_TIER_ALIAS, new String[] {mTestDirPath}, TEST_TIER_CAPACITY_BYTES, "");
+        TEST_TIER_ALIAS, new String[] {mTestDirPath}, TEST_TIER_CAPACITY_BYTES, TEST_WORKER_DIR);
```

Comment:
Previously by setting the property to "" basically made the extra directory nonexistent. However if it is unset (null) the default value for the alluxio worker property key gets used now instead adding that extra directory. It essentially is testing the same functionality, but the previous test made the condition check simpler by passing "" because it was a valid value at the time.

Reply:
So before it put a block in a location...something like `/var/tmp/ad786das8sd/{some_identifier}/block` but if they set the property key to anything else it would've been `/var/tmp/ad786das8sdblock/{block_dir}/{some_identifier}/block` Because they set the `block_dir` to the empty string it wasn't in the directory path. It shouldn't be possible to do that any more.

Reply:
Got it, thanks for explaining. Seems like the original configuration being tested isn't something that we actually want the user to set. @apc999 can you confirm?